session_0/README.md

```diff
@@ -17,6 +17,8 @@ Rails. You can refer to books on programming or websites like
 
 > Read the [installation guide](/installation.md) and [setting up local
 > workspace](/essential_git.md) to get started.
+>
```

Comment:
Remove changes from session 0, 1 and 2

Reply:
This time as confirmed by you, I've used git add session_3 only. But still It's showing this, I don't know why :(

Reply:
Now, I got it why it's showing session_0 and session_1 files also. It's because we're applying changes to the same pull request so 'Files changed' section will show all of the files changed till now. 
But if you look at the latest commit link 'Implemented Session 3', there you'll find only the session_3 not anything else :)

![image](https://user-images.githubusercontent.com/64270972/105049434-a5e67100-5a92-11eb-8c27-ff5eaad9e100.png)

Reply:
@abhishekkumar2718 can you review my PR

Reply:
Yeah, I reviewed it already and added the `Approved` tag

Reply:
Ohk thank you :)